test/Spec/MLabsPAB/Contract.hs

```diff
@@ -62,7 +62,7 @@ sendAda :: Assertion
 sendAda = do
   let txOutRef = TxOutRef "e406b0cf676fc2b1a9edb0617f259ad025c20ea6f0333820aa7cef1bfe7302e5" 0
       txOut = TxOut (Ledger.pubKeyHashAddress pkh1) (Ada.lovelaceValueOf 1250) Nothing
-      initState = def {utxos = [(txOutRef, txOut)]}
+      initState = def {_utxos = [(txOutRef, txOut)]}
```

Comment:
Seeing a lot of uses of the `_` accessors, lets use lenses wherever we can.
So something like
`initState = def & utxos .~ [(txOutRef, txOut)]`